spec/spec_helper.rb

```diff
@@ -1,6 +1,9 @@
 require "simplecov"
+
 SimpleCov.start do
   add_filter "initializers/git_revision"
+  add_filter "v0/example_controller"
+  add_filter "v0/sessions_controller"
```

Comment:
if we're going to put these filters in to prevent simplecov from picking up these files for coverage **temporarily** maybe we should put a TODO: or FIXME: comment in there as well so we know to remove this in the future, or else we might end up thinking we have 90% coverage in the future when we don't? Just a nitpick as I know this PR is just a WIP to get us moving along.